src/asm/gamepad.s

```diff
@@ -187,34 +183,30 @@ SoftReset:
   sta GameMode
   pla
   pla
+  nop
+  nop
+  nop
   rts
```

Comment:
Can we skip the `nop`s and just `rts` right away?

Alternatively, we could shrink the `.reloc` routine (making it easier to place) by moving the `lda` outside of it, and thus still be able to use the three bytes here.

Something like
```
.org $cbd3
  lda $0780
  jsr LoopWildWarpIndex
  lda WildWarpLocations,y
  ; ...
  pla
  rts

.reloc
LoopWildWarpIndex:
  clc
  tay
  bpl +
  ; ...
```

Does this also allow patching less code here?  I'd expect the tail end of this routine probably isn't changed much (but I don't have the disasm handy to compare right now).

Reply:
I tried to rts right away, but there are other pieces of ASM nearby that jump directly to that rts as an early return, so either we need to edit more ASM, or that rts needs to stay in place.

Reply:
If you move the `lda` out of the subroutine then it should keep the same alignment for the `rts`, but make better use of those three bytes.

Reply:
Done and done.